proposals/lip-0071.md

```diff
@@ -20,28 +20,29 @@ This LIP is licensed under the [Creative Commons Zero 1.0 Universal](https://cre
 
 ### Rewards Proportional to the Validators Weight
 
-* This module distributes the remaining stake rewards i.e. `stakeRewardActiveValidators` proportional to the validators weight. It is given to the active validators (per round) after giving the minimum reward. For the Lisk mainchain, as calculated in the [`table`](#reward-computation), the rewards are `90.9*10^8` beddows or `90.9 LSK`. This way, the module achieves fairness (higher weight implies higher rewards) while ensuring that rewards of low-weight validators do not go below the threshold of profitability.
-* In principle, if percentage of rewards shared equally among active validators is set to `0%` i.e.`FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS == 0` (as described in [table](#specification)), then it will be a proof-of-stake system where rewards are proportional to the validator weight. On the other hand, if the percentage of rewards shared equally among active validators is set to `100%` i.e. `FACTOR_MINIMUM_REWARD_ACTIVE_VALIDATORS == 10000`, it will be the previous proof-of-stake (PoS) system where each validator received the same reward for forging a block irrespective of their weights. Therefore, this reward system can be seen as a generalization of the previous one and allows flexibility to developers to use the variant of Proof-of-Stake they find more appropriate for their use case.
+* This module distributes the remaining staking rewards proportional to the validators weight. It is given to the active validators that generate a block, on top of the minimum reward. For the Lisk mainchain, the rewards distributed according to stake are 90% of total active validator rewards, assuming `101` active validators. Therefore `90.9*10^8` beddows or `90.9 LSK` per round are normally distributed to active validators. This way, the module achieves fairness (higher weight implies higher rewards) while ensuring that rewards of low-weight validators do not go below the threshold of profitability.
+* Technically, each time an active validator generates a block they get a fraction of stake reward (depending on the current number of active validators), proportionally to their BFT weight. Note that an active validator might generate multiple blocks in a round (assuming no missed blocks): this could happen in case the total number of validators of a round are less than the expected one (which is `roundLength`). In such cases, the total reward paid out in a round could slightly exceed the estimate of `defaultReward * roundLength`, in case validators with higher than average weight generate more blocks than the others. However, since in our PoS systems the validators are randomly shuffled, the expected value of the total reward given in a round is not higher than `defaultReward * roundLength`.
```

Comment:
To be 100% precise, perhaps we should replace `defaultReward` by `getDefaultRewardAtHeight()` and `roundLength` by `Pos.getRoundLength()`. But since we are in the rationale, I am not sure this is 100% necessary.